tools/crm_resource.c

```diff
@@ -1860,7 +1819,7 @@ main(int argc, char **argv)
             break;
 
         case cmd_list_agents:
-            rc = list_agents(out, options.agent_spec);
+            rc = pcmk__list_agents(out, options.agent_spec);
```

Comment:
Will we end up showing errors twice, once in the function and once in `pcmk__output_and_clear_error()`? See `done:`

Reply:
Yes and no:
```
[root@rhel9-cluster-1 ~]# crm_resource --list-agents xyz
No agents found for standard 'xyz'
crm_resource: Error performing operation: Error occurred
```

Reply:
I suppose that's fine then. Though I would expect "no such object" in this particular example ...

Reply:
We could certainly revisit the error codes here, too.  "Error occurred" is really unhelpful, and I suppose that's coming from just returning pcmk_rc_error.  That's what list_agents was doing and I just copied it over.

Reply:
In the off chance someone's script is checking for exit status 1, let's hold that for a compatibility break, but I think it's a good idea